spec/models/post_spec.rb

```diff
@@ -489,6 +493,37 @@ def perform_update
                post.activity_pub_actor.activities.where(attrs).size
             ).to eq(2)
           end
+
+          context "when the acting user is different from the post user" do
```

Comment:
I am not 100% sure I follow the test acse here. When revising a post, we want to create an activity for the `acting_user`, but when creating posts, it should be ignored, correct?

Reply:
Yes, these tests are only needed for Update activities. The activity actor and the object attributedTo (aka the user used by the PostCreator) will always be the same actor (/person/user).